src/backend/marsha/bbb/management/commands/update_pending_sessions.py

```diff
@@ -0,0 +1,29 @@
+"""For each pending classroom, get their meeting infos:
+ update learning analytics, and end it if needed."""
+from django.core.management import BaseCommand
+
+from marsha.bbb.models import Classroom
+from marsha.bbb.utils.bbb_utils import get_meeting_infos
+
+
+class Command(BaseCommand):
```

Comment:
maybe add in the file name the `classroom` keyword ? Otherwise we don't know what is pending reading running jobs